openai/table_openai_completion.go

```diff
@@ -23,7 +24,7 @@ func tableOpenAiCompletion(ctx context.Context) *plugin.Table {
 		},
 		Columns: []*plugin.Column{
 			// Top columns
-			{Name: "completion", Type: proto.ColumnType_STRING, Transform: transform.FromField("Text"), Description: "Completions for a given text prompt."},
+			{Name: "completion", Type: proto.ColumnType_STRING, Transform: transform.FromField("Completion"), Description: "Completions for a given text prompt."},
```

Comment:
I thought so because of the  "completion" !=  "Completion" (notice the capital letter)

Reply:
@nicocesar, We use the [DefaultTransform](https://github.com/turbot/steampipe-plugin-openai/blob/main/openai/plugin.go#L16) function in plugin level, it will help us to convert the `UpperCamelCase` to `snake_case` by default. If we have a property called `URL` or any lower-level properties like(`Completion.Text`) in this case we need to use the transform function in the column.